test suite: bump MAXIMUM_INGEST_LOCK_DURATION

I'm attempting to run the integration test suite in a Docker OverlayFS in a Xen virtual machine running on a RAID5 array of 5400RPM Barracudas, and can consistently reproduce this test failure:

```
Failures: [ERROR] InReviewWorkflowIT.testCuratorSendsCommentsToAuthor:244 
Failed test if Ingest Lock exceeds max duration scripts/search/data/tabular/50by1000.dta.zip
```
I propose bumping the value of MAXIMUM_INGEST_LOCK_DURATION, currently 3 seconds, to 15 to match MAXIMUM_PUBLISH_LOCK_DURATION.

If IQSS would consider accepting my subsequent pull request, my integration test suite runs will succeed.

--- a/src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java
@@ -53,7 +53,7 @@ public class UtilIT {
     private static final String API_TOKEN_KEY = "apiToken";
     private static final String BUILTIN_USER_KEY = "burrito";
     private static final String EMPTY_STRING = "";
-    public static final int MAXIMUM_INGEST_LOCK_DURATION = 3;
+    public static final int MAXIMUM_INGEST_LOCK_DURATION = 15;
     public static final int MAXIMUM_PUBLISH_LOCK_DURATION = 15;
     
     private static SwordConfigurationImpl swordConfiguration = new SwordConfigurationImpl();

